### PR TITLE
Eig jax linalg

### PR DESCRIPTION
**Pull request by rubythalib33**
#6084

### CODE REVIEW

**Comment by jkeane508** [comment]
Hey, just re-ran the tests as they didn't run properly :)

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Hey, you have seemed to implement the function and the test in two different sub modules hence why tests are failing. 

Your test is in `jax.lax.linalg` instead of `jax.numpy.linalg`

**Comment by rubythalib33** [comment]
Sure I'll change it thank you

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py [hunk 1]
```diff
@@ -38,6 +38,30 @@ def test_jax_numpy_det(dtype_and_x, as_variable, native_array, num_positional_ar
...
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=False,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend="jax",
+        fn_tree="numpy.linalg.eig",
+        rtol=1e-04,
+        atol=1e-04,
+        a=np.asarray(x[0], dtype=dtype[0]),
```
Comment (jkeane508) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py:61]: we actually don't use `np.asarray` anymore for inputs, you can just remove this

**Review by jkeane508 — CHANGES_REQUESTED** [review]
Some changes needed here, also the test is failing for this but it may be an issue with our backends.

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py (hunk 1 above)
Comment (rubythalib33) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_linalg.py:61]: sure thank you

**Comment by rubythalib33** [comment]
thank you

**Pull request merged**